dashboard/app/views/devise/shared/_oauth_links.haml

```diff
@@ -27,3 +27,8 @@
 :javascript
 
   $('.oauth_sign_in').click(dashboard.clientState.reset);
+  // If the makerBridge is available, we are in the Maker App, which has it's
```

Comment:
super nit: its

Reply:
https://media.giphy.com/media/vwI4mYEHP8k0w/giphy.gif
This is such a bad habit of mine. Thanks for catching!